src/Geno/C++/GUI/MainWindow.cpp

```diff
@@ -210,6 +218,9 @@ bool MainWindow::BeginFrame( void )
 	if( pMenuBar->ShowTextEdit          ) pTextEdit         ->Show( &pMenuBar->ShowTextEdit );
 	if( pMenuBar->ShowOutputWindow      ) pOutputWindow     ->Show( &pMenuBar->ShowOutputWindow );
 
+	bool ivsb_show = true;
```

Comment:
Prefer verbose names over obscure abbreviations.

That said, the status bar should always be shown so there's no need to pass in a boolean here.